mender-auth/client/client.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <mender-auth/client/client.hpp>
```

Comment:
Again my question from a couple days before -- what's the plan here? The code lives under *mender-auth* and then we link it to *mender-update* as well?

Reply:
No, that's not how mender-update get's the auth token.

This is done via IPC